src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- This list of runtimes needs to match the set of runtimes specified in eng/workloads/workloads.csproj -->
-    <RuntimeIdentifiers>win-x86;win-x64;win-arm64;linux-x64;linux-arm64;osx-x64;osx-arm64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64;linux-x64;linux-arm64;linux-musl-x64;osx-x64;osx-arm64</RuntimeIdentifiers>
```

Comment:
This LGTM but I actually think that later we should consider having the dashboard pack being a single package that is just framework-dependent as opposed to self-contained and needing for us to target all of these RIDs specifically.